Source/Spray/SprayProperties.H

```diff
@@ -47,6 +47,10 @@ getInpVal(
 {
   for (int spf = 0; spf < SPRAY_FUEL_NUM; ++spf) {
     const std::string var_read = fuel_names[spf] + "_" + varname;
+    if (!ppp.contains(var_read)) {
+      amrex::Abort(
+        "particles." + var_read + " must be specified in the input file.");
+    }
```

Comment:
Is this needed? An error should be raised by `ppp.get` if the requested variable is not found.